src/etc/inc/util.inc

```diff
@@ -946,7 +946,7 @@ function log_error($error)
 * RESULT
 *   null
 ******/
-function log_msg($msg, $prio = LOG_ERR)
+function log_msg($msg, $prio = LOG_WARNING)
```

Comment:
I think we should get this one single line in now rather than later.

Reply:
I don't see any problems ) do I have to make a pr for it or will you make the changes yourself and I'll just keep that in mind when updating this one closer to 22.7?

Reply:
if you have the time please make a PR, better to retain authorship if we can :)

Reply:
sure, thanks)